modules/packages/manifests/setup.pp

```diff
@@ -375,6 +375,11 @@
                     options      => ['arch=amd64'],
                     gpg_id       => '0EBFCD88',
                     gpg_key      => 'puppet:///modules/packages/0EBFCD88.txt';
+                'nrpe':
+                    url_path     => 'repos/apt/custom/nrpe',
+                    distribution => $::lsbdistcodename,
+                    options      => ['trusted=yes'],
```

Comment:
can we remove the "trusted=yes" option?

Reply:
all repos are set with trused=yes options